doc/source/python_reference.rst

```diff
@@ -139,7 +139,7 @@ Functions:
    read_bem_solution
    read_bem_surfaces
    read_cov
-   read_dip
+   read_dipoles
```

Comment:
>
> Well I've already changed the API a bit to return a single dict instead of a tuple of pos, orientation, amplitude, etc. It will be very easy to make it a simple object with attributes, so I'll at least do that.
>
>
so Dipole class would simply inherit from dict with a repr + cplot method.

can you isolate the change in a new PR?